src/frontend/src/pages/PIAIntakeForm/index.tsx

```diff
@@ -212,7 +212,10 @@ const PIAFormPage = () => {
     try {
       if (buttonValue === 'submit') {
         const updatedPia = await upsertAndUpdatePia({
-          status: PiaStatuses.MPO_REVIEW,
+          status:
+            pia?.hasAddedPiToDataElements === false
+              ? PiaStatuses.MPO_REVIEW
+              : PiaStatuses.INCOMPLETE,
         });
```

Comment:
I guess you do not need to call this if the status is already INCOMPLETE and hasAddedPiToDataElements is true/null. Seems like an extra call.

Reply:
We would still need to do a check before submit right? There is no need for else, but for a delegated review we have to change the status.

Reply:
Sure, explicit is fine 👍 I was just hoping if we could avoid an extra call if the status is already what it is - in case of non-delegated flow.